testing/src/main/java/org/oppia/testing/TestCoroutineDispatcher.kt

```diff
@@ -10,17 +9,26 @@ import kotlinx.coroutines.Runnable
 import kotlinx.coroutines.test.DelayController
 import kotlinx.coroutines.test.UncompletedCoroutinesError
 import java.lang.Long.max
+import java.util.*
```

Comment:
Avoid Wildcard import

Reply:
Nice catch--done. Android Studio is a bit too eager to use wildcard imports. :(

Reply:
Thanks